### DISCUSSION TITLE
20 Claim Threshold Determination

### DISCUSSION THREAD

**Discussion by MMath19103** [CMSgov/price-transparency-guide#275] [Q&A] [answered]
@shaselton  I've reviewed the discussion within the following 'OON Allowed-Amounts File Where does 20 Claim Threshold Apply #181'
We'd like feedback on the interpretation we have settled upon as it is not clear from the answer in #181 

o	For each entity (company or SF customer) and plan
o	Within the time period specified (current date – 6-months, 90-days of claim data from that date): 

1.	Extract all claims paid to an out-of-network provider (i.e., a provider we have no contract with)
          a.	Facility and professional claims
2.	Eliminate claims that are out of scope (e.g., Medicare, Medicaid, CHIP, etc.)
3.	Evaluate remaining claims at the claim level when institutional; at the line level when professional
4.	Count the number of times the primary procedure or billing code + code set version has been billed
5.	Sort the subset by Allowed amount
Determine if there are >= 20 occurrences of the primary procedure code + code set version + allowed amount in the set of claims
*When true, capture the Place of Service (when professional), NPI and TIN of OON providers that were paid for that service at that allowed-amount and report on MRF#2
*When false, do not report
*When no claims exist for the reporting entity + Plan - create the 0 record "file"

Can we confirm or correct with the maintainers that this is in line with what was intended?

Thank you!

**Comment by shaselton-usds** [comment, marked as answer]
Hi @MMath19103 

Sorry for the delay in response. We actually updated the "Technical Clarifications" sections for the TiC site (https://www.cms.gov/healthplan-price-transparency/resources/technical-clarification) hopefully to help out questions around the 20-claim threshold.

It looks like you're pretty much on-point with your steps. I would point out that steps 4 and 5 of "code set version" isn't necessary when counting towards the 20 claims. (Although, I suspect that the code set versions wouldn't have much of an impact on whether some service/item crosses that threshold...). Also, for #5, the combination including the allowed amounts doesn't need to reach a threshold of 20 before reporting -- it is *only* the billing code that triggers the 20 claim threshold. You have the ability to report different allowed amounts and ultimate billed charges for the allowed amounts.

Let me know if you have any further questions or if this response was confusing.

**Reply by MMath19103** [reply]
Thank you.
